app/src/js/modules/liveeditor.js

```diff
@@ -122,11 +128,22 @@
                                     win.getSelection().getRangeAt(0).insertNode(doc.createTextNode(content));
                                 }
                             }
-                        }).on('keypress', function (e) {
-                            return e.which != 13;
-                        }).on('focus blur', function (e) {
-                            $(this).html($(this).text());
                         });
+
+                        if(fieldType == 'textarea') {
+                            $(this).on('keypress', function (e) {
+                                if(e.which == 13) {
+                                    e.preventDefault();
+                                    doc.execCommand('insertHTML', false, '<br><br>');
+                                }
+                            });
+                        } else {
+                            $(this).on('keypress', function (e) {
+                                return e.which != 13;
+                            }).on('focus blur', function (e) {
+                                $(this).html($(this).text());
```

Comment:
This should probably be executed for textarea types as well because it undoes HTML other html changes incurred by the browser. For example, hitting `ctrl/cmd + B` adds `<strong>` tags around the selected text.